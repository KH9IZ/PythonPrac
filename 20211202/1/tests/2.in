class C(metaclass=dump):
    ...
class D(C):
    def f(self, q, qq='QKRQ'):
        print(q + qq)
d = D()
d.f('- Hi!\n- ')
