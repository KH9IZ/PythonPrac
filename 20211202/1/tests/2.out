f: ('- Hi!\n- ',), {}
- Hi!
- QKRQ
